orchestrator/careplancontributor/service.go

```diff
@@ -128,6 +128,12 @@ func (s Service) RegisterHandlers(mux *http.ServeMux) {
 		log.Info().Msgf("Redirecting to %s", s.frontendUrl)
 		http.Redirect(response, request, s.frontendUrl, http.StatusFound)
 	})
+
+	// Logout endpoint
+	mux.HandleFunc(basePath+"/zorgplatform/logout", func(response http.ResponseWriter, request *http.Request) {
+		s.SessionManager.Destroy(response, request)
+		http.Redirect(response, request, s.frontendUrl, http.StatusOK)
+	})
```

Comment:
I see the `zorgplatform` is defined in the issue, but it feels like we either:

1. Want a generic logout, no matter how you launched orca, OR
2. this logic should be moved into the zorgplatform `service.go`

I'd prefer option 1, maybe the `zorgplatform` can be replaced with a wildcard?

Reply:
I'm not 100% convinced, since the IDP sends (or should sent, according to SAMLv2 spec) a signed `LogoutRequest`, which' signature we _could_. I don't think it's required, but maybe in future?

Reply:
This would be future implementation right?

Reply:
I think we can do with 1 logout URL (having it on `/logout`?), since we don't need to check the signature right now. And session destroying isn't adapter-specific (especially if we can just redirect back to the Referer URL).